daemon/qrexec-client.c

```diff
@@ -831,6 +847,8 @@ int main(int argc, char **argv)
                 FD_SET(wait_connection_end, &read_fd);
                 select(wait_connection_end+1, &read_fd, NULL, NULL, 0);
             }
+        } else if (local_cmdline == NULL) {
+            abort();
```

Comment:
This is wrong on many levels here: `local_cmdline` can be legally NULL in this case (it's basic qrexec dom0 -> VM call with data over stdio). But even if it wouldn't, `local_cmdline` isn't used after this check, so it's pointless here.